### PR TITLE
Add Joan as docs maintainer

### PR DESCRIPTION
**Pull request by alexandratran**
This PR adds Joan Edwards (@joaniefromtheblock) to the list of Besu docs maintainers. She has authored the following PRs:

- #1485 
- #1486 
- #1490 
- #1501 
- #1504 
- #1502 
- #1503 

You can see [her current list of PRs](https://github.com/hyperledger/besu-docs/pulls?q=is%3Apr+author%3Ajoaniefromtheblock+).

> Note: This PR also moves Mike Sanko to emeritus status.

### CODE REVIEW

**Comment by joaniefromtheblock** [comment]
Thanks @alexandratran! I accept this nomination and hope to be a long-term contributor. Thank you very much for the nomination

**Review by bgravenorst — APPROVED** [review]
lgtm

**Review by jflo — APPROVED** [review]
Joan asks good questions and does good work.

**Review by gfukushima — APPROVED** [review]
lgtm

File: MAINTAINERS.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-| Alexandra Tran   | alexandratran  | alexandratran  |
-| Byron Gravenorst | bgravenorst    | bgravenorst    |
-| Madeline Murray  | MadelineMurray | madelinemurray |
-| Mike Sanko       | mjsmike62      | mjsmike62      |
+| Name             | GitHub             | LFID               |
+|------------------|--------------------|--------------------|
+| Alexandra Tran   | alexandratran      | alexandratran      |
+| Byron Gravenorst | bgravenorst        | bgravenorst        |
+| Madeline Murray  | MadelineMurray     | madelinemurray     |
+| Joan Edwards     | joaniefromtheblock | joaniefromtheblock |
```
Comment (macfarla) [inline MAINTAINERS.md:13]: nit: should be one line up to keep the list sorted alphabetically

**Review by macfarla — APPROVED** [review]
LGTM - thanks Joan!

**Review by jframe — APPROVED** [review]
LGTM

**Review by siladu — APPROVED** [review]

**Review by fab-10 — APPROVED** [review]

**Comment by alexandratran** [comment]
Merging this as it's been 2 weeks.

**Pull request merged**